app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
It strikes me that the _biggest_ difference (that would require the most mangling) between g-cloud's search and the way we want to do dos's filtering is g-cloud's desire to treat lots quite separately, affecting the other options offered, while dos's treatment of lots is quite straightforward. At some point if we want to bring the two into line together this will have to be resolved, and that seemed a bit much for a two-point story.